schema/manifest_backwards_compatibility_test.go

```diff
@@ -110,14 +110,14 @@ func TestBackwardsCompatibilityManifestList(t *testing.T) {
 			fail: false,
 		},
 	} {
-		sum := sha256.Sum256([]byte(tt.manifest))
+		sum := sha256.Sum256([]byte(tt.manifestlist))
```

Comment:
That is not what I said. I meant that if you're in the process of cleaning up, it would be better to use the go-digest package.

I've filed https://github.com/opencontainers/image-spec/issues/513 to take care this, so don't worry about it.